src/default_panels/StyleAxesPanel.js

```diff
@@ -113,7 +125,12 @@ class StyleAxesPanel extends Component {
           </Section>
           <TraceTypeSection
             name={_('Range')}
-            traceTypes={['choropleth', 'scattergeo']}
+            traceTypes={[
```

Comment:
could we get rid of this tracetype section and merge it into the one above somehow? I think enough has changed/we've learned enough since we added this hack that we can do better now :)